README.md

```diff
@@ -20,6 +28,27 @@ pnpm dev
 
 Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
 
+### Troubleshooting
+If you encounter any issues during setup, refer to the following troubleshooting tips:
+
+- For new contributors you may encounter the error
+  ```
+   GITHUB_PAT is not configured in the environment.
+   Request quota exhausted for request POST /graphql
+  ```
+  In place of running `pnpm dev` create you own GitHub access token. [Read Steps here](https://docs.github.com/en/enterprise-server@3.9/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens)
+  Then run the following command instead:
+  ```
+  GITHUB_PAT=<YOUR_KEY> pnpm dev
+  ```
+
+- If you are getting the error:
+  ```
+   fatal: unable to access 'https://github.com/coronasafe/leaderboard-data.git/': Could not resolve host: github.com
+   Error executing command: git clone
+  ```
+  Make sure you are connected to an active internet connection, or your antivirus may also cause this issue. Try running the command with your antivirus closed.
```

Comment:
Should we mention this? I doubt if they can clone this repo itself if contributor has these issues. It could also be because of some networks blocking github too, there are far too many reasons than the ones mentioned for this to happen.

Reply:
When I cloned the main repo there was no issue with the installation and running. 
But when I tried running pnpn install. It was showing an error for cloning the data repo. Only after I closed my antivirus the download was successful.
I think this should be mentioned or if you want we can remove it.

Reply:
I have removed it